### ISSUE TITLE
The difference between the results and the kitti

### ISSUE THREAD

**Issue by xjjs** [caizhongang/waymo_to_kitti_converter#3]
Hi,
Using the converted results of part of waymo, I train and test the PointRCNN project
1.training set: training_0000 and training_0001
2.testing set: training_0002

The training loss is normal, which is about 1.0~1.5, but the mAP is weird, which is < 1.0. 
Considering the format of the waymo_kitti_results is the same as that of kitti, It's strange to get such a poor performance.

**Comment by caizhongang** [comment]
The two datasets are very different, not only in terms of the format, but the nature of the data. For example, in KITTI, the surface is mostly flat, but there are many slopes in Waymo. Hence, the voxel generation may be strongly affected. 

Also, the point cloud coordinate systems are different. In KITTI, it is the coordinate system of the top lidar, which is >1m above the ground. But in KITTI, since there are multiple LiDARs, the common practice is to use the self-driving car's frame, which is at the ground.

I would recommend tuning the hyperparameters. Like the range of voxels etc.

You may use the tools/kitti_label_visualizer.py to confirm the differences between the converted Waymo dataset and the KITTI dataset. The visualization tools are in the midst of development as we speak but they should work properly at the moment.

**Comment by xjjs** [comment]
@caizhongang Thanks for your detailed explanations, the visualization results of tools/kitti_label_visualizer.py is hard to be recognized.
![waymo](https://user-images.githubusercontent.com/32333197/82535856-1b074c80-9b7a-11ea-97f3-ae1212e71121.png)
![kitti](https://user-images.githubusercontent.com/32333197/82535865-1fcc0080-9b7a-11ea-856b-b9ad059e8d07.png)

**Comment by xjjs** [comment]
@caizhongang  Hi, I have visualized the data with the tool: https://github.com/kuixu/kitti_object_vis, and it shows that the converted results are good on flat surface. And the poor performance may come from the labels,  I used the labels of all lidars but only the data of top and front lidars

**Comment by caizhongang** [comment]
Hi, great to hear that! Yes, KITTI only labels the area visible to the front camera. In this case, you may want to use image_0/ and label_0/. At the moment, the calib/ files are all for the front camera. 

For the visualization tool, you can actually drag and scroll to change your viewing angles.
